marklogic-data-hub/src/main/java/com/marklogic/hub/cli/command/VerifyDhfInDhsCommand.java

```diff
@@ -92,16 +99,17 @@ private void verifyModules() {
             DocumentMetadataHandle metadata = documentManager.readMetadata("/com.marklogic.hub/config.sjs", new DocumentMetadataHandle());
             DocumentMetadataHandle.DocumentPermissions perms = metadata.getPermissions();
 
-            Set<DocumentMetadataHandle.Capability> capabilities = perms.get("flowDeveloper");
-            verify(capabilities.contains(DocumentMetadataHandle.Capability.READ), "Flow developer should be able to read modules");
-            verify(capabilities.contains(DocumentMetadataHandle.Capability.INSERT), "Flow developer should be able to insert modules");
-            verify(capabilities.contains(DocumentMetadataHandle.Capability.EXECUTE), "Flow developer should be able to execute modules");
+            Set<DocumentMetadataHandle.Capability> capabilities = perms.get("data-hub-module-reader");
```

Comment:
@rjrudin can you please explain the change here and for flowOperator?

Reply:
The command is now using a permissions string of data-hub-module-reader,read,data-hub-module-reader,execute,data-hub-environment-manager,update,rest-extension-user,execute .

Reply:
We need a meeting with the DHS team to determine if they want to keep the flowOperator and flowDeveloper roles. If so, they'll need to adapt to inherit these new roles.